css/user.css

```diff
@@ -0,0 +1,58 @@
+.user-body {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: auto;
+}
+
+.user-body-simple {
+    border: 1px solid var(--color-hover-light-grey);
+    border-radius: 0.2rem;
+    padding: 2rem;
+    margin: 1.5rem auto;
+    width: 25%;
+}
+
+.user-avatar {
+    display: flex;
+    justify-content: center;
+}
+
+.user-avatar-img {
+    width: 250px;
+    height: 250px;
+}
```

Comment:
you can use relative units after 10px. But it ok you can change it later.

Reply:
I'll keep that in mind. Thank you!